assets/aml-benchmark/components/src/aml_benchmark/batch_output_formatter/result_converters.py

```diff
@@ -103,6 +103,8 @@ def convert_result_ground_truth(self, result: Dict[str, Any]) -> Dict[str, Any]:
                 use_ground_truth_input = True
         elif self._model.is_aoai_model():
             use_ground_truth_input = True
+        elif self._model.is_vision_oss_model():
```

Comment:
What is the logic around ground_truth_input ? When is it used ? Do you see a case for vision models to have this support ?

Reply:
It seems to only be used for quality pipeline. The ground truth is passed into the batch-inference-preparer which hashes it and then it is outputted by the batch-output-formatter. batch-output-formatter will process the output from the scoring script, but does not actually use the ground_truth information. It is a pass through used for quality pipeline